x/cellarfees/migrations/v1/keeper/abci.go

```diff
@@ -0,0 +1,52 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	"github.com/peggyjv/sommelier/v8/app/params"
+)
+
+// BeginBlocker emits rewards each block they are available by sending them to the distribution module's fee collector
+// account. Emissions are a constant value based on the last peak supply of distributable fees so that the reward supply
+// will decrease linearly until exhausted.
+func (k Keeper) BeginBlocker(ctx sdk.Context) {
```

Comment:
I am confused why this is running during a migration.... and what the semantics of this are....